charts/schema-registry/README.md

```diff
@@ -0,0 +1,56 @@
+# Schema Registry Helm Chart
+
+Installs [Pravega Schema Registry](https://github.com/pravega/schema-registry) atop Kubernetes.
+
+## Introduction
+
+This chart bootstraps a [schema registry](https://github.com/pravega/schema-registry) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+  - Kubernetes 1.15+
+  - Helm 3+
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```
+$ helm install my-release charts/schema-registry
+```
+
+This command deploys schema registry on the Kubernetes cluster with default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```
+$ helm uninstall my-release
+```
+
+This command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following table lists the configurable parameters of the schema registry chart and their default values.
+
+| Parameter | Description | Default |
```

Comment:
Please use naming guidelines described here: https://github.com/pravega/pravega/wiki/Configuration-Naming-Guidelines for all new configurations anywhere in Pravega. For example: `replica.count` not `replicaCount`.  For some examples, see: https://github.com/pravega/pravega/blob/master/documentation/src/docs/security/pravega-security-configurations.md